server/athenian/api/align/queries/metrics.py

```diff
@@ -171,49 +209,89 @@ async def _calculate_team_metrics(
     calculator = make_calculator(account, meta_ids, mdb, pdb, rdb, cache)
     tasks = []
     if pr_metrics:
-        pr_participants = [{
-            PRParticipationKind.AUTHOR: team,
-        } for team in _loginify_teams(teams, prefixer)]
-        tasks.append(calculator.calc_pull_request_metrics_line_github(
-            pr_metrics, time_intervals, quantiles, [], [], [repos], pr_participants,
-            LabelFilter.empty(), JIRAFilter.empty(), True, account_bots,
-            release_settings, logical_settings, prefixer, branches, default_branches, False,
-        ))
+        pr_participants = [
+            {
+                PRParticipationKind.AUTHOR: team,
+            }
```

Comment:
Instead of minimizing the number of lines, it produced an ugly beast.

Reply:
Golang enforces egyptian brackets everywhere, and since we are going to write Go at some point, the goal is to minimize the friction between the two. Let me investigate how `go fmt` does it, because hacking Black is an order of magnitude easier than the standard tooling.

Reply:
the problem was the trailing comma, required by flake8-commas, now it looks better